packages/ui/src/components/Dropdown/DropdownTrigger.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+import { ComponentPropsWithoutRef, forwardRef, ReactNode } from 'react';
+import { useDropdownContext } from './Dropdown.context';
+import { dropdownTrigger } from './Dropdown.css';
+
+export type DropdownTriggerProps = {
+  children?: ReactNode;
+} & ComponentPropsWithoutRef<'div'>;
+
+export const DropdownTrigger = forwardRef<HTMLDivElement, DropdownTriggerProps>(
+  ({ children, className = '', ...props }, ref) => {
+    const { setIsOpen, value, placeholder, triggerRef } = useDropdownContext();
+    const handleClick = () => {
+      setIsOpen((prev) => !prev);
+    };
+
+    return (
+      <div
+        ref={(node) => {
+          if (typeof ref === 'function') {
+            ref(node);
+          } else if (ref) {
+            (ref as any).current = node;
+          }
+          if (triggerRef) {
+            (triggerRef as any).current = node;
+          }
+        }}
```

Comment:
```suggestion
 ref={(node) => {
          if (typeof ref === 'function') {
            return ref(node);
          }
          if (ref !== null && ref !== undefined) {
            return (ref.current = node);
          }
          if (triggerRef) {
            return triggerRef.current = node;
          }
        }}
```

이렇게 하시고 

DropdownContextValue interface에서 

```tsx
  triggerRef: MutableRefObject<HTMLElement | null>;
```

로 하면 깔끔하게 할 수 있어요!